packages/renderer/src/lib/dialogs/MessageBox.svelte

```diff
@@ -25,6 +25,8 @@ const showMessageBoxCallback = async (options?: MessageBoxOptions) => {
   message = options.message;
   if (options.detail) {
     detail = options.detail;
+  } else {
+    detail = null;
```

Comment:
I guess you mean undefined and not null ?

Reply:
Yes, old Java habits die hard 😉. Fixed.